src/layouts/AlertLayout.tsx

```diff
@@ -2,15 +2,42 @@ import { LabelProps, RendererProps } from "@jsonforms/core"
 import { Alert } from "antd"
 import { AlertLayoutOptions } from "../ui-schema"
 import { withJsonFormsLabelProps } from "@jsonforms/react"
+import ReactMarkdown from "react-markdown"
 
 export function AlertLayout({ text, uischema }: LabelProps & RendererProps) {
-  const options = uischema.options as AlertLayoutOptions
+  const options = (uischema?.options ?? { style: {} }) as AlertLayoutOptions
```

Comment:
it'd make this more runtime safe if you were to cast uischema instead:
> const options = (uischema as LabelLayoutUISchema<unknown>).options
ie if AlertLayoutOptions gets a required property, this cast will still steamroll it, allowing the code to later do something like
```
options.newRequiredProperty.OopsThisWillBeATypeError
```